src/screens/PluginStore/PluginStore.tsx

```
@@ -0,0 +1,614 @@
+
+  // Pagination handlers
+  const handleChangePage = (
+    event: React.MouseEvent | null,
+    newPage: number,
+  ) => {
+    setPage(newPage);
+  };
+  const handleChangeRowsPerPage = (
+    event: React.ChangeEvent<{ value: string }>,
+  ) => {
+    setRowsPerPage(parseInt(event.target.value, 10));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)